capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -22,16 +21,7 @@ const BookSearchTile = (props) => {
           </Col>
           <Col md="auto">
             <Container className="center-vertical">
-              <DropdownButton id="dropdown-list-add" className="dropdown-add" title="Add to List">
-                {
-                  props.userBookLists.map(bookList =>
-                    <Dropdown.Item key={bookList.id}
-                      onSelect={() => addBookToBookList(props.book.id, bookList.id)}>
-                      {bookList.name}
-                    </Dropdown.Item>
-                  )
-                }
-              </DropdownButton>
+              <BookListAddDropdown userBookLists={props.userBookLists} book={props.book}></BookListAddDropdown>
```

Comment:
nit: Same here - make this a self-closing tag if there are no children.

Reply:
Fixed!